.github/workflows/ci.yml

```diff
@@ -1,11 +1,13 @@
+name: "CI"
+
 on: [push, pull_request]
 
 jobs:
   build:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go: [ '1.19', '1.18', '1.17' ]
+        go: [ '1.17', '1.18', '1.19', '1.20' ]
```

Comment:
Because we haven't specified non-parallel execution these will all run at the same time, but if we disable or limit parallel execution later (or if any fork does, etc) then the reversing of the order here will have impact. Was this reversing intentional? Do we want to run older versions first in those scenarios? I would suggest we want to run newer versions first.

Reply:
Is the order guaranteed? The resolver currently uses 1.17 - https://github.com/awslabs/amazon-ecr-containerd-resolver/blob/master/go.mod#L3 so maybe it makes sense either way

Reply:
Fail fast is not enabled so all jobs will run. Order should have not impact here.

Reply:
@sparr , @turan18 removed this from this PR to unblock adding Go 1.20 support and upgrade of the linter in the CI pipeline. Perhaps we can break that off into a seperate thread or leave as is.